### ISSUE TITLE
A11y_SQLAzureDataStudioMAC_Run query_VoiceOver: When the user runs the code inside 'New query' section, the Voice-Over does not announce the output message explicitly to the users.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/azuredatastudio#19872]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudioMAC_June2022;#SQL Azure Data Studio;#MAC;#FTP;#DesktopApp;#VoiceOver;#AINotInScope;#WCAG4.2.12;

## Environment Details:
Application Name: SQL Azure Data Studio
Application Version: Version 1.37.0
Mac Version: macOS Monterey 12.4
Screen Reader: Voice-Over

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch SQL Azure data studio.
3. Connect to a server.
4. Tab to 'File' menu.
5. Tab to 'New Query' and hit return key.
6. Type a SQL code.
7. Tab till 'Run' button.
8. Turn on Voice-Over and observe. 

## Actual:
When the user runs the code inside 'New query' section, the Voice-Over does not announce the output message explicitly to the users.

## Expected:
When the user runs the code inside 'New query' section, the Voice-Over should announce the output message explicitly to the users.

## User Impact:
The Voice-Over users will face difficulty in knowing the important output message or notification if not announced by the assistive technologies.

## Attachment:
![11_Message output](https://user-images.githubusercontent.com/98393966/176432523-a7bc176a-62b6-4201-878c-3ef668d2daf9.gif)

**Comment by alanrenmsft** [comment]
the issue has been fixed in the latest insider build.

**Comment by alanrenmsft** [comment]
please let us know if the fix is not working. closing since the SLA is near.

**Comment by sspatil123** [comment]
![image](https://user-images.githubusercontent.com/85863999/220017612-5b270f53-e2dc-487c-a4d0-b11e309a59e7.png)
GithubTags:#Closed;Version 1.41.2
